Resources/public/js/views/services/ez-serversideviewservice.js

```diff
@@ -161,6 +166,33 @@ YUI.add('ez-serversideviewservice', function (Y) {
         },
 
         /**
+         * Fires notify event basing on node.
+         * the app is not yet a bubble target of the view service,
```

Comment:
you didn't get [my previous comment](https://github.com/ezsystems/PlatformUIBundle/pull/263#discussion_r33790718) right. This comment is about an implementation detail so it should not be in the doc block of the method but just above the corresponding code as inline comment (with `//`)